keps/sig-node/3085-pod-conditions-for-starting-completition-of-sandbox-creation/README.md

```diff
@@ -1251,6 +1290,14 @@ checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
 
+Checking the Pod conditions on nodes with this feature enabled is the simplest way to check if the feature is enabled properly on a vanilla k8s cluster.
```

Comment:
This question isn't totally relevant for this feature, since this is an administrator-enabled feature controlled via kubelet flag, not something the user controls with an API server resource spec.

Reply:
Got it - thanks - I will remove this section with the above note.

Reply:
Done